src/pages/Blog/index.tsx

```
@@ -1,16 +1,36 @@
-import { Input, Tag, Badge, Avatar, Empty } from 'antd'
+import { Link, useNavigate } from 'react-router-dom'
+import { Input, Tag, Avatar, Empty, Button, Modal } from 'antd'
 import { SearchOutlined, EyeOutlined } from '@ant-design/icons'
 import styled from 'styled-components'
 import { useDispatch, useSelector } from 'react-redux'
-import { selectBlogInfo, selectTagInfo } from '@/store/modules/global/selector'
+import { selectBlogInfo, selectTagInfo, selectUserInfo } from '@/store/modules/global/selector'
 import ROUTES from '@/utils/config/routes'
+import { style } from '@/theme'
+
+interface Tag {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Tag'. Consider to delete it or rename it.

'Tag' is defined here:


(lint/suspicious/noRedeclare)